shared/profile/search.desktop.js

```diff
@@ -0,0 +1,58 @@
+// @flow
+import React from 'react'
+import ServiceFilter from '../searchv3/services-filter'
+import ResultsList from '../searchv3/results-list'
+import UserInput from '../searchv3/user-input'
+import {Box, Icon, Text} from '../common-adapters'
+import {globalStyles, globalColors, globalMargins} from '../styles'
+
+import type {Props} from './search'
+
+const Search = (props: Props) => (
+  <Box style={styleSearchContainer}>
+    <Box style={{...styleSearchRow}}>
+      <Box style={{...globalStyles.flexBoxRow, flexGrow: 1}}>
+        <Box style={{flexGrow: 1}}>
+          <UserInput
+            userItems={props.userItems}
+            showAddButton={props.showAddButton}
+            onRemoveUser={props.onRemoveUser}
+            onClickAddButton={props.onClickAddButton}
+            placeholder={props.placeholder}
+            usernameText={props.searchText}
+            onChangeText={props.onChangeText}
```

Comment:
Now that my user input arrow keys commit is in, this needs no-ops for the handlers to pass flow: https://github.com/keybase/client/commit/eedb9739243d9c087f5519dfe8d0ef1369d7c350

I made these handlers required props so we wouldn't forget them. For native they never get called, and can be null or no-ops.